app/Http/Controllers/CalculatorController.php

```diff
@@ -23,65 +23,46 @@ public function __construct(Request $request)
 
         $fileName = $request->input('fileName');
 
-        if (!isset($fileName)) { 
+        if (!isset($fileName)) {
             throw new \Exception("Please input a Sura name");
         }
 
-        $suraFile = File::get(storage_path('suras/'.$fileName));
+        $suraFile = File::get(storage_path('suras/' . $fileName));
 
         if (!isset($suraFile)) {
             throw new \Exception("Sura file not found");
         }
 
         $this->fullSura = new FullSura($suraFile);
-        $this->fullSura->name = $fileName;
+        $this->fullSura->Name = $fileName;
     }
 
     public function mapSura()
     {
-        $this->fullSura->numberOfVerses = $this->fullSura->calculateNumberOfVerses();
-        $this->fullSura->numberOfWords = $this->fullSura->calculateNumberOfWords();
-        $this->fullSura->numberOfLetters = $this->fullSura->calculateNumberOfLetters();
-        
-        $this->fullSura->wordOccurrences = $this->counter->countWordsInString($this->fullSura->suraString);
-        $this->fullSura->wordIndex = $this->indexer->indexWordsInString($this->fullSura->suraString);
-        
-        $this->fullSura->letterOccurrences = $this->counter->countLettersInString($this->fullSura->suraString);
-        $this->fullSura->LettersMapIndexes = $this->indexer->indexLettersInString($this->fullSura->verses);
+        $this->fullSura->NumberOfVerses = $this->fullSura->calculateNumberOfVerses();
```

Comment:
used `UpperCase` format for all returned fields to have a unified standard, we can also consider `underscore_case` instead